sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/KeyVaultAccessControlClient.java

```diff
@@ -0,0 +1,180 @@
+package com.azure.security.keyvault.administration;
+
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.Context;
+import com.azure.security.keyvault.administration.implementation.models.KeyVaultErrorException;
+import com.azure.security.keyvault.administration.implementation.models.RoleAssignment;
+import com.azure.security.keyvault.administration.implementation.models.RoleAssignmentProperties;
+import com.azure.security.keyvault.administration.implementation.models.RoleDefinition;
+
+/**
+ * The {@link KeyVaultAccessControlClient} provides synchronous methods to view and manage Role Based Access for the
+ * Azure Key Vault. The client supports creating, listing, updating, and deleting {@link RoleAssignment}.
+ * Additionally, the client supports listing {@link RoleDefinition}.
+ */
+@ServiceClient(builder = KeyVaultAccessControlClientBuilder.class)
+public class KeyVaultAccessControlClient {
+    private final KeyVaultAccessControlAsyncClient asyncClient;
+
+    /**
+     * Creates an {@link KeyVaultAccessControlClient} that uses a {@code pipeline} to service requests
+     *
+     * @param asyncClient The {@link KeyVaultAccessControlAsyncClient} that the client routes its request through.
+     */
+    KeyVaultAccessControlClient(KeyVaultAccessControlAsyncClient asyncClient) {
+        this.asyncClient = asyncClient;
+    }
+
+    /**
+     * Get the vault endpoint URL.
+     *
+     * @return The vault endpoint URL.
+     */
+    public String getVaultUrl() {
+        return asyncClient.getVaultUrl();
+    }
+
+    /**
+     * Get all role definitions that are applicable at scope and above.
+     *
+     * @param scope   The scope of the role definition.
+     * @param context Additional context that is passed through the HTTP pipeline during the service call.
+     * @return A {@link PagedIterable} of {@link RoleDefinition role definitions}.
+     * @throws KeyVaultErrorException if the operation is unsuccessful.
+     * @throws NullPointerException   if the {@code scope} is null.
+     */
+    public PagedIterable<RoleDefinition> listRoleDefinitions(String scope, Context context) {
```

Comment:
Suggestion: For .net we added a helper extensible enum to make it easier for to choose between the possible valid scopes ( `/`, `/keys`, and `/keys/<someKeyId>`. You may want to implement something for all the string `scope` parameters.

Reply:
Sounds good to me, thanks for the suggestion!